frontend/src/components/UserMessage.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import styled from 'styled-components'
+
+import { LogoutButton } from './LogoutButton'
+
+export const UserMessage = () => {
+    const [message, setMessage] = useState("");
+
+    const accessToken = useSelector((store) => store.user.login.accessToken);
+    const userId = useSelector((store) => store.user.login.userId);
+
+    const AUTH_URL = `https://thessan-rebeka-auth-api.herokuapp.com/sessions/${userId}/userMessage`;
+
+    fetch(AUTH_URL, {
+        method: "GET",
+        headers: { Authorization: `${accessToken}` },
+    })
+        .then((response) => response.json())
+        .then((json) => {
+            setMessage(json);
+        });
+    return (
+        <>
+            <Message>{message}</Message>
```

Comment:
Nice and clear message when logging in! It only shows on the first login though, tried loggin in twice. Perhaps you guys can have a look at that :)

Reply:
We can't see this error you're referring too. We've tried to login several times in a row and it works ☺️